pom.xml

```diff
@@ -300,6 +300,14 @@
             <activation>
                 <jdk>1.7</jdk>
             </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>com.facebook.presto</groupId>
+                    <artifactId>presto-jdbc</artifactId>
+                    <version>0.212</version>
+                    <!-- <scope>test</scope> -->
```

Comment:
It looks like we have been including jdbc drivers only for the test phase.  Is there any difference for Presto?